torchao/_models/llama/eval.py

```diff
@@ -122,9 +125,16 @@ def run_evaluation(
         else:
             if not TORCH_VERSION_AT_LEAST_2_5:
                 unwrap_tensor_subclass(model)
+        if "float8wo" in quantization:
+            quantize_(model, float8_weight_only())
+        if "float8dq" in quantization:
+            quantize_(model, float8_dynamic_activation_float8_weight())
+        if "float8saq" in quantization:
```

Comment:
nit: maybe just `float8sq`? to be consistent with `dq`

Reply:
Added it for testing, removed it for now. Static quant will be a slightly different approach